src/journal/bsod.c

```diff
@@ -0,0 +1,249 @@
+/* SPDX-License-Identifier: LPGL-2.1-or-later */
+
+#include <errno.h>
+#include <getopt.h>
+#include <linux/vt.h>
+#include <sys/ioctl.h>
+#include <stdio.h>
+
+#include "alloc-util.h"
+#include "build.h"
+#include "fd-util.h"
+#include "fileio.h"
+#include "io-util.h"
+#include "log.h"
+#include "main-func.h"
+#include "qrcode-util.h"
+#include "sd-id128.h"
+#include "sd-journal.h"
+#include "sysctl-util.h"
+#include "terminal-util.h"
+
+static void help(void) {
+        printf("%s\n\n"
+               "filters the journal to fetch the first message from the\n"
+               " current boot with an emergency log level and displays it"
```

Comment:
missing newline character

Reply:
and extra space at the beginning

Reply:
> and extra space at the beginning

Oh...I just broke the string to two lines quite hastily.